No unknown public methods test failing

Running Centos 7 perl 5.16.3
I ran into this repackaging an RPM on Centos 7.

```# make test 'TEST_FILES=xt/*.t' RELEASE_TESTING=1
PERL_DL_NONLAZY=1 /usr/bin/perl "-MExtUtils::Command::MM" "-e" "test_harness(0, 'blib/lib', 'blib/arch')" xt/*.t
xt/api.t ..... 1/2931 # Found undocumented method 'filename' defined at 'PPI::Document::filename'

#   Failed test 'No unknown public methods in 'PPI::Document''
#   at /usr/share/perl5/vendor_perl/Test/ClassAPI.pm line 189.
#          got: '1'
#     expected: '0'
# Found undocumented method 'filename' defined at 'PPI::Document::filename'

#   Failed test 'No unknown public methods in 'PPI::Document::Fragment''
#   at /usr/share/perl5/vendor_perl/Test/ClassAPI.pm line 189.
#          got: '1'
#     expected: '0'
# Found undocumented method 'type' defined at 'PPI::Statement::Sub::type'

#   Failed test 'No unknown public methods in 'PPI::Statement::Sub''
#   at /usr/share/perl5/vendor_perl/Test/ClassAPI.pm line 189.
#          got: '1'
#     expected: '0'
xt/api.t ..... 2383/2931 # Looks like you failed 3 tests of 2931.
xt/api.t ..... Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/2931 subtests 
xt/author.t .. ok     
xt/meta.t .... ok   
xt/pmv.t ..... ok       

Test Summary Report
-------------------
xt/api.t   (Wstat: 768 Tests: 2931 Failed: 3)
  Failed tests:  143, 209, 933
  Non-zero exit status: 3
Files=4, Tests=3297, 51 wallclock secs ( 0.26 usr  0.01 sys + 50.48 cusr  0.13 csys = 50.88 CPU)
Result: FAIL
Failed 1/4 test programs. 3/3297 subtests failed.
make: *** [test_dynamic] Error 255
```

--- a/xt/api.t
+++ b/xt/api.t
@@ -6,7 +6,7 @@ use lib 't/lib';
 use PPI::Test::pragmas;
 use Test::More;
 BEGIN {
-        my $tests = 2931 + ($ENV{AUTHOR_TESTING} ? 1 : 0);
+        my $tests = 2935 + ($ENV{AUTHOR_TESTING} ? 1 : 0);
 	if ( $ENV{AUTOMATED_TESTING} or $ENV{RELEASE_TESTING} ) {
 		plan( tests => $tests );
 	} else {
@@ -314,6 +314,7 @@ prototype=method
 block=method
 forward=method
 reserved=method
+type=method
 
 [PPI::Statement::Scheduled]
 PPI::Statement::Sub=isa
@@ -402,6 +403,7 @@ flush_locations=method
 normalized=method
 complete=method
 errstr=method
+filename=method
 STORABLE_freeze=method
 STORABLE_thaw=method
 

